canonical-web-and-design/vanilla-framework
# Update list examples to use list elements

## Done

- Ensure all list examples use list elements

Fixes #4225 

## QA

- Open [demo](https://vanilla-framework-4302.demos.haus/)
- Check responsive divider lists still work as expected

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
